reconcile/utils/oc.py

```diff
@@ -1207,6 +1195,43 @@ def is_kind_supported(self, kind: str) -> bool:
 OCClient = Union[OCNative, OCDeprecated]
 
 
+class OCLocal:
+    def __init__(
+        self,
+        cluster_name,
+        jh=None,
+        settings=None,
+        insecure_skip_tls_verify=False,
+    ):
+        self.cluster_name = cluster_name
+        oc_base_cmd = ["oc", "--kubeconfig", "/dev/null"]
+        if insecure_skip_tls_verify:
+            oc_base_cmd.extend(["--insecure-skip-tls-verify"])
+
+        self.jump_host = None
+        if jh is not None:
+            self.jump_host = JumpHostSSH(jh, settings=settings)
+            oc_base_cmd = self.jump_host.get_ssh_base_cmd() + oc_base_cmd
+
+        self.oc_base_cmd = oc_base_cmd
+
+    def process(self, template, parameters=None):
+        if parameters is None:
+            parameters = {}
+        parameters_to_process = [f"{k}={v}" for k, v in parameters.items()]
+        cmd = [
+            "process",
+            "--local",
+            "--ignore-unknown-parameters",
+            "-f",
+            "-",
+        ] + parameters_to_process
+        result = OCDeprecated._run(
```

Comment:
Calling a hidden method in another class is a little odd, even though it looks like a static one.  Should process perhaps be part of that OCDreprecated class?

Reply:
A potential solution would be to have the new `OCLocal` inherit from `OCDeprecated` in the same fashion as `OCNative` currently does.